benchmarks/imagenet/resnet50/simclr.py

```diff
@@ -85,9 +85,9 @@ def configure_optimizers(self):
             # Square root learning rate scaling improves performance for small
             # batch sizes (<=2048) and few training epochs (<=200). Alternatively,
             # linear scaling can be used for larger batches and longer training:
-            #   lr=0.3 * self.batch_size * self.trainer.world_size / 256
+            #   lr=0.3 * self.batch_size / 256
```

Comment:
There's a slight problem introduced by this: For certain models we need to know the batch size per GPU in the initializer (e.g. for the size of the queue in SwaV). Do you have a suggestion how to address this?

Reply:
Hmm, we could also use the `devices` parameter.

Reply:
I think it would be the cleanest approach. And as soon as we have the world size we can calculate the global batch size.